src/components/ha-selector/ha-selector-duration.ts

```diff
@@ -18,6 +19,19 @@ export class HaTimeDuration extends LitElement {
 
   @property({ type: Boolean }) public required = true;
 
+  protected firstUpdated(changedProps: PropertyValues): void {
```

Comment:
I am thinking now. Should this be in will update?

Reply:
`ha-form` applies the default value if nothing is set. It's weird that we can get here and then set the duration.

I think instead in the rendering we should just do `const value = this.value || "0:00:00"`

Reply:
We need to send the value to HA form

Reply:
Ha form doesnt default the value.

Not for selectors

Reply:
Fix it here https://github.com/home-assistant/frontend/blob/dev/src/components/ha-form/compute-initial-ha-form-data.ts#L3